src/app/fsat/explore-opportunities/explore-opportunities-form/fan-data-form/fan-data-form.component.ts

```diff
@@ -125,9 +127,21 @@ export class FanDataFormComponent implements OnInit {
     this.calculate();
   }
 
-  checkWarnings(){
+  checkWarnings() {
     this.specifiedError1 = this.fsatWarningService.checkFanWarnings(this.fsat.fanSetup).fanEfficiencyError;
     this.specifiedError2 = this.fsatWarningService.checkFanWarnings(this.fsat.modifications[this.exploreModIndex].fsat.fanSetup).fanEfficiencyError;
+    if (this.fsat.fanSetup.drive == 4) {
```

Comment:
Shouldn't need to add this logic here. This check is already being done in `fsatWarningsService` 

`this.specifiedError1` and `this.specifiedError2` will have `specifiedDriveEfficiencyError` member variable.